.mvn/jvm.config

```diff
@@ -0,0 +1 @@
+-Xmx2048m
```

Comment:
@jtnord just manually checked the last successful builds on ci.j.o, and indeed the max I saw has been [731 MB](https://jenkins.ci.cloudbees.com/job/core/job/jenkins-core/4504/console).

So I guess putting at least something like 800 MB would be reasonnable to prevent the GC to chime in possibly more often that we'd like and slow down the build.

WDYT?